doc/ca-in-ofc.rst

```diff
@@ -0,0 +1,144 @@
+    &= (\textbf{A}\vec{x})^{T}\textbf{C}^{T}\textbf{C}\textbf{A}\vec{x} \\
+    &= \vec{x}^{T}\textbf{A}^{T}\textbf{C}^{T}\textbf{C}\textbf{A}\vec{x}.
+    \end{align*}
+
+This means :math:`textbf{Q} = \textbf{A}^{T}\textbf{C}^{T}\textbf{C}\textbf{A}`.
+The matrix :math:`\textbf{C}^{T}\textbf{C}` is chosen to be:
+
+.. math::
+    \textbf{C}^{T}\text{C} \equiv \left( \frac{2\pi}{\lambda(\ell)} \right)^{2} \text{diag}(\vec\alpha)^{2},
+
+where :math:`\vec{\alpha}` is the alpha value of normalized point source sensitivity (PSSN) in the basis :math:`\mathbb{Z}`.
+:math:`\lambda(\ell)` is the effective wavelength under different filter band :math:`\ell`.
+
+We can rewrite the matrix :math:`\textbf{F}` as:
+
+.. math::
+
+    \begin{align*}
+    \textbf{F} &= \text{inv}\left(\sum_{i}w_{i}\textbf{Q}_{i} + \rho^{2}\textbf{H}\right) \\
+    &= \text{inv}\left( \left( \frac{2\pi}{\lambda(\ell)} \right)^{2} \sum_{i}w_{i} (\textbf{A}^{T}\text{diag}(\vec\alpha)^{2}\textbf{A})_{i} + \rho^{2}\text{diag}(\vec{\rho}_{h}\vec{h}^{T})^{2}\right) \\
+    &= \text{inv}\left (\left( \frac{2\pi}{\lambda(\ell)} \right)^{2} \sum_{i}w_{i}\textbf{M}_{i}^{T}\text{diag}(\vec\alpha)_{i}^{2}\textbf{M}_{i} + \rho^{2}\text{diag}(\vec{\rho}_{h}\vec{h}^{T})^{2}\right).
+    \end{align*}
+
+The offset :math:`\vec{u}` is:
+
+.. math::
+
+    \begin{align*}
+    \vec{u} &= - \textbf{F}\sum_{i}w_{i}(\textbf{Q}\vec{x}_{k})_{i} \\
+    &= - \left( \frac{2\pi}{\lambda(\ell)} \right)^{2} \textbf{F} \sum_{i}w_{i} \textbf{M}_{i}^{T}\text{diag}(\vec\alpha)_{i}^{2}\textbf{M}_{i} \vec{x}_{k, i}.
+    \end{align*}
+
+Consider the correction of wavefront error between center and corcer, we rewrite the above equation as:
+
+.. math::
+
+    \begin{align*}
+    \vec{u} &= - \left( \frac{2\pi}{\lambda(\ell)} \right)^{2} \textbf{F} \sum_{i}w_{i} \textbf{M}_{i}^{T}\text{diag}(\vec\alpha)_{i}^{2} (\textbf{M}_{i} \vec{x}_{k, i} +\vec{y}_{2, i}) \\
+    &= - \left( \frac{2\pi}{\lambda(\ell)} \right)^{2} \textbf{F} \sum_{i}w_{i} \textbf{M}_{i}^{T}\text{diag}(\vec\alpha)_{i}^{2} \vec{m}_{x, i}.
+    \end{align*}
+
+Consider three different reference points "x0, "0", and "x00". "x0" means the offset will only trace the previous one; "0" means the offset will trace the real value and target for 0; "x00" means the offset will only trace the relative changes of offset without regarding the real value.
+Assume the telescope's state is :math:`\vec{s}_{0}` in the time :math:`k=0` and :math:`\vec{s}` in time :math:`k`, and we can intentionally shift :math:`\vec{m}_{x}` to be:
+
+.. math::
+
+    \begin{align*}
+    \vec{m}_{x} &\rightarrow \vec{m}_{x}, \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \  \text{for x0} \\
+    &\rightarrow \vec{m}_{x} + \rho^{2}\textbf{H}\vec{s}, \ \ \ \ \ \ \ \ \ \ \ \ \ \ \ \text{for 0} \\
+    &\rightarrow \vec{m}_{x} + \rho^{2}\textbf{H}(\vec{s} - \vec{s}_{0}), \ \ \ \text{for x00}.
```

Comment:
We could try to align the  \text{for x0}, \text{for 0}, and \text{for x00} to make them look better on lsst.io.